saleor/settings.py

```diff
@@ -610,8 +610,3 @@ def get_host():
     'RELAY_CONNECTION_ENFORCE_FIRST_OR_LAST': True,
     'RELAY_CONNECTION_MAX_LIMIT': 100
 }
-
-CHECKOUT_PAYMENT_GATEWAYS = {
-    DUMMY: pgettext_lazy('Payment method name', 'Dummy gateway'),
```

Comment:
Why did you remove everything though?

It was fine like it was before (see below) until #3250 is resolved.
```python
CHECKOUT_PAYMENT_GATEWAYS = {
     DUMMY: pgettext_lazy('Payment method name', 'Dummy gateway')}
```